src/app/messages-page/create-access-group/create-access-group.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="modal-header">
+  <div class="modal-header-x" (click)="bsModalRef.hide()">
+    <i-feather name="x"></i-feather>
+  </div>
+</div>
+<div class="likes-modal-container">
+  <header class="border-bottom p-2">
+    <h1 class="fs-17px font-weight-bold">Create Group Chat</h1>
+  </header>
+  <form class="p-2" (submit)="this.createGroupChat($event)">
```

Comment:
should I remove `this.` ?

Reply:
it's harmless to leave it. It's superfluous since angular binds to it anyway so it doesn't change anything functionally either way.